wallet/src/Pos/Wallet/Web/Methods/Backup.hs

```diff
@@ -46,7 +46,11 @@ restoreWalletFromBackup WalletBackup {..} = do
                           & each . _2 %~ \(AccountMetaBackup am) -> am
 
             addSecretKey wbSecretKey
-            -- XXX Transaction
+            -- My current thinking is that we don't want a transaction here. If we're restoring
```

Comment:
A minor style note: I think the style guide is such that we should be talking not directly in first person, but I do appreciate this has been left here just for ease of review, really 😀 

Ps. Minor typo below, that's "backup", not "bacup" 😀